tests/serialized_test_data_generation/submit_job.sh

```diff
@@ -1,12 +1,54 @@
 #!/bin/bash
 
+# stop on all errors
+set -e
+
+# setup run directory
 RUNDIR=/rundir
+cp /FV3/fv3.exe $RUNDIR/fv3.exe
+
+# setup directory for serialized data
 TEST_DATA_DIR=$RUNDIR/test_data
+mkdir -p $TEST_DATA_DIR
+
+# setup environment
+
+# setup run environment
+cd $RUNDIR
 NUM_RANKS=`python ${RUNDIR}/count_ranks.py $RUNDIR/fv3config.yml`
 ulimit -s unlimited
-cp /FV3/fv3.exe $RUNDIR/fv3.exe
-mkdir -p $TEST_DATA_DIR
+
+# run the model (deduce MPI configuration from config file)
+# note: we want to catch errors, hence the set +e/-e logic
+echo ">>> Running model, inspect stdout.out / stderr.out in rundir for details"
+set +e
+mpirun -l -np $NUM_RANKS $RUNDIR/fv3.exe 2>$RUNDIR/stderr.out 1>$RUNDIR/stdout.out
+if [ $? -ne 0 ] ; then
+    echo ">>> Error occurred while running the model"
+    echo ">>>>> cmd:"
+    echo "mpirun -l -np $NUM_RANKS $RUNDIR/fv3.exe"
+    echo ">>>>> stderr:"
+    cat $RUNDIR/stderr.out
+    echo ">>>>> stdout:"
+    cat $RUNDIR/stdout.out
+    echo ">>>>> Aborting"
+    exit 1
```

Comment:
nice! would it also help to echo the env, or would that be too much?

Reply:
Done.